package.json

```diff
@@ -366,6 +366,11 @@
           "type": "boolean",
           "default": true,
           "description": "Specifes whether OmniSharp should use VS Code editor settings for C# code formatting (use of tabs, indentation size)."
+        },
+        "omnisharp.format.enable": {
```

Comment:
ok, what about omnisharp.useEditorFormattingSettings? Why format.enable should be at csharp, but format options about tabs / new lines - at omnisharp namespace?